lib/app/modules/teamAttendance/views/team_attendance_view.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import 'package:get/get.dart';
+import 'package:managment_app/Themes/themes.dart';
+
+import '../controllers/team_attendance_controller.dart';
+
+class TeamAttendanceView extends GetView<TeamAttendanceController> {
+
+  @override
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          leading: IconButton(
+            icon: Icon(Icons.arrow_back_rounded),
+            onPressed: () {},
+          ),
+          backgroundColor: Color(0xFF002431),
+          title: Text('MY TEAM',
+              style: TextStyle(
```

Comment:
instead of assigning seperate style to text you can use **style: Get.theme.kTitleTextStyle** this i have already assinged the fontfamily and style use the required textstyle from there **(check themes.dart)**